src/Components/Overlay.js

```diff
@@ -46,15 +50,16 @@ const Overlay = () => {
     }
   }
 
-  if (activeStep < 224) {
-    console.log(true);
+  if (activeStep === 224) {
+    reachExit();
   }
 
   return (
     <div className="overlay-container">
       <div className="overlay-tile-grid">
-        <TileContainer activeTiles={activeStep} />
-        <Exit activeStep={activeStep}/>
+        {levelComplete === false && <TileContainer activeTiles={activeStep} />}
```

Comment:
I like this component.  If we can find a way to move the logic out of the return it might help.